docs/docs/spec.md

```diff
@@ -146,16 +141,18 @@ A nullness operator is one of 4 values:
 >     -   One way to conceptualize this is that `String NO_CHANGE` means
 >         "non-null `String`" but that `T NO_CHANGE` means "nullness comes from
 >         the value of `T`."
-> -   `UNSPECIFIED`: This is the operator produced by not putting `@Nullable` on
->     a type usage *in code that is outside a [null-marked scope]*. Roughly, it
->     is the operator assigned to "completely unannotated code."
+> -   `UNSPECIFIED`: This is the operator produced for a type usage *outside* of
+>     [null-marked scope]*, where nullness type-use annotations are applicable
+>     **FIX THAT?**
+>     yet none appears. Roughly, it is the operator assigned to "completely
+>     unannotated code."
 >     -   The type usage `String UNSPECIFIED` includes `"a"`, `"b"`, `"ab"`,
->         etc., but the developer did not specify whether to include `null`.
+>         etc., but it has been left ambiguous whether to include `null`.
```

Comment:
I like the "developer did not specify" phrasing for its precision. But presumably my approach of beating them over the head with variants of the word "specified" is passing up the opportunity to add _something_, so you're presumably aiming to do that with "ambiguous?"

Reply:
I could be okay with it, but it felt strange for a spec like this to personify "the developer" like this. All it really knows is what's in the code.

They generally say to avoid the passive-voice-with-unspecified-subject, but I always wondered at that advice, because sometimes the point is that the subject absolutely doesn't matter!

Reply:
I've tweaked this. Can go back to "the developer" if you prefer (I just don't, that's all).